app/src/interfaces/useSurveyApi.interface.ts

```diff
@@ -358,11 +359,17 @@ export type IEditSurveyRequest = IGeneralInformationForm &
   ISurveyLocationForm &
   IProprietaryDataForm &
   IUpdateAgreementsForm & { partnerships: IGetSurveyForViewResponsePartnerships } & ISurveySiteSelectionForm &
-  IParticipantsJobForm;
+  IParticipantsJobForm &
+  ISurveyBlockForm;
 
 export interface IGetSampleSiteResponse {
   sampleSites: IGetSampleLocationRecord[];
 }
+
+export interface IGetSampleSiteByIdResponse {
```

Comment:
I would argue this interface is a bit extraneous, considering it isn't any different from `IGetSampleSiteResponse`, and it's not being used anywhere in the codebase.